bin/tidyall

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env perl
+#
+# Tidyall command with perltidy version constraint.
+use Perl::Tidy;
+
+use constant REQUIRED_VERSION => '20230912';
+my $detected_version = $Perl::Tidy::VERSION;
+my @tidyall_argv = @ARGV;
+
+sub is_force_flag { $_ eq '--force' }
+
+unless ($detected_version == REQUIRED_VERSION) {
+    print  STDERR "Incorrect version of perltidy.\n";
+    printf STDERR "- Detected: %s\n+ Required: %s\n\n", $detected_version, REQUIRED_VERSION;
+
+    my $force_run = grep { is_force_flag } @ARGV;
+
+    unless ($force_run) {
+        printf STDERR "Please install the appropriate version of perltidy.\n";
+        printf STDERR "If you want to proceed anyways, re run with --force flag.\n";
```

Comment:
Maybe this can be formatted to make it pop up on GH workflows (https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message)

Reply:
This is a local-only tool, `tools/tidy` is not run in any workflow that I can find, `perlcritic` does through `run_unit_tests.sh` and Makefile targets.

The idea is that is a transparent wrapper over `tidyall` so no special text processing is needed on ever divergent bash scripts.